test/solvers/utilities/tree_graphs.jl

```diff
@@ -0,0 +1,23 @@
+import Graphs as gr
+import NamedGraphs as ng
```

Comment:
For the sake of keeping the style consistent throughout the library I would prefer:
```julia
using Graphs: add_edge!, ...
using NamedGraphs: NamedGraph, ...
```
If we don't stay consistent with the style it leads to chaos where code is written in different styles and it is a lot of work to go back and make it consistent later.

Reply:
Yes, I meant to change these to the current style of ITensorNetworks. I will do that and in any other files I find.